velox/exec/tests/AggregationTest.cpp

```diff
@@ -3707,4 +3707,33 @@ TEST_F(AggregationTest, nanKeys) {
       {makeRowVector({e0, e1}), e1});
 }
 
+TEST_F(AggregationTest, spillDuringDistinct) {
```

Comment:
Can we just improve the existing distinctWithSpilling instead of adding a new one? Thanks!